app/helpers/application_helper.rb

```diff
@@ -13,7 +13,11 @@ def item_status_check(item)
   end
 
   def format_price_per_unit(item)
-    "#{number_to_currency(item.price_per_unit)} / lb "
+    if item.class.to_s == "Item"
```

Comment:
Since both paths have all the same string except the price method called, I think we should extract this.